src/commands/project/create-variable-group.test.ts

```diff
@@ -297,37 +204,19 @@ describe("setVariableGroupInBedrockFile", () => {
 
   test("Should pass adding a valid variable group name when bedrock file exists with empty variableGroups", async () => {
     // Create random directory to initialize
-    const randomTmpDir = path.join(os.tmpdir(), uuid());
-    fs.mkdirSync(randomTmpDir);
-
-    // create bedrock file to simulate the the use case that `spk project init` ran before
-    const bedrockFileData: IBedrockFile = {
-      rings: {},
-      services: {},
-      variableGroups: []
-    };
-
-    const asYaml = yaml.safeDump(bedrockFileData, {
-      lineWidth: Number.MAX_SAFE_INTEGER
-    });
-
-    fs.writeFileSync(path.join(randomTmpDir, "bedrock.yaml"), asYaml);
+    const randomTmpDir = createBedrockYaml();
 
     await setVariableGroupInBedrockFile(randomTmpDir, variableGroupName);
 
-    const bedrockFilePath = path.join(randomTmpDir, "bedrock.yaml");
-    expect(fs.existsSync(bedrockFilePath)).toBe(true);
-
-    const bedrockFile = readYaml<IBedrockFile>(bedrockFilePath);
+    expect(isBedrockFileExists(randomTmpDir)).toBe(true);
+    const bedrockFile = readBedrockFile(randomTmpDir);
```

Comment:
In relation to above comment, these two lines are expecting a file in random dir. May be randomDir needs to be passed.

Reply:
sorry I do not get it. Can you elaborate?